net_http/server/internal/evhttp_server.cc

```diff
@@ -405,4 +405,8 @@ bool EvHTTPServer::EventLoopSchedule(std::function<void()> fn) {
   return result == 0;
 }
 
+void EvHTTPServer::ScheduleReply(std::function<void()> fn) {
+  server_options_->executor()->Schedule(fn);
```

Comment:
why does this become safe now, i.e. scheduled outside the event loop?

Reply:
It is mentioned everywhere that libevent does not handle deffered replies well:
1. https://github.com/libevent/libevent/issues/944
2. https://github.com/azat/libevent/commit/5ff8eb26371c4dc56f384b2de35bea2d87814779. 

It is a simpler solution to reply inline.

Reply:
Thanks! I don't think I know libevents any more :)

Reply:
LGTM.

Reply:
Can you please approve it as well?